### PR TITLE
nodemon and live-server should be required from node_modules

### PR DESCRIPTION
**Pull request by Razzeee**

### CODE REVIEW

**Comment by opvasger** [comment]
Hello Razzeee!

according to [NPM documentation](https://docs.npmjs.com/cli/install) this isn't a problem
>By default, npm install will install all modules listed as dependencies in package.json.
>
>With the --production flag (or when the NODE_ENV environment variable is set to production), npm will not install modules listed in devDependencies.

And both modules are listed as development-dependencies in the `/example/package.json` file

**Comment by Razzeee** [comment]
yes, they will, but they won't be available on the path. so running the `sh` file will fail.
Or well getting the command and running it by hand, as you have to do on windows due to it not understanding sh files and the use of `&`.
Still it's not on the path, so it won't work.

**Comment by opvasger** [comment]
ah yes, thanks for the correction! I think moving the script into the `example/package.json` is the right thing to do in this case - if you agree, I'll merge the change once you make the change :sunny:

**Comment by Razzeee** [comment]
Agreed, just did not want to force your sh approach out from under your feet :)

I changed everything that is needed. Please have a look if it still works fine for you. Windows seems happy.

File: example/package.json [hunk 1]
```diff
@@ -2,8 +2,12 @@
   "name": "example",
   "version": "0.1.0",
   "devDependencies": {
+    "concurrently": "^4.1.0",
```
Comment (opvasger) [inline example/package.json]: should be removed

File: example/package.json [hunk 2]
```diff
@@ -2,8 +2,12 @@
...
+    "elm": "0.19.0-bugfix2",
     "live-server": "1.2.1",
-    "nodemon": "1.18.7",
-    "elm": "0.19.0-bugfix2"
+    "nodemon": "1.18.9"
+  },
+  "scripts": {
+    "watch": "concurrently \"nodemon --watch ../src --watch src --ext elm --exec \\\"elm make src/Main.elm --output=dist/index.js\\\"\" \"live-server --watch=./dist\""
```
Comment (opvasger) [inline example/package.json]: should be renamed to "start" and run the one-liner from `examle/dev.sh`. `example/dev.sh` should then be deleted.

File: readme.md [hunk 3]
```diff
@@ -10,8 +10,7 @@ If you want to try it, you can clone the repo and run this `bash`-script from th
 ```bash
 npm install
 
-chmod +x ./dev.sh
+npm run watch
```
Comment (opvasger) [inline readme.md]: should be changed to `npm start`

File: example/package.json (hunk 2 above)
Comment (Razzeee) [reply example/package.json]: `&` doesn't exist on windows. Should windows then use another command or won't it be supported?

**Comment by opvasger** [comment]
The rationale for removing `concurrently` is that:
1. I don't care about Windows compatability
2. I want as few node_modules as possible

I'm also pretty sure that you can still easily run the script using git-bash, but I'm not sure about this :sunny:

both `example/package-lock.json` and `example/dev.sh` should be deleted.

File: example/package.json (hunk 2 above)
Comment (opvasger) [reply example/package.json]: `1.0.0` won't contain any of this code :sunny: alternatively, write another script that uses [this](https://www.windows-commandline.com/cmd-start-command/)

Comment (opvasger) [reply example/package.json]: I recommend `start:win`

**Comment by opvasger** [comment]
I'll add the changes for you on the `master` branch, so just pull my changes 👍

**Pull request closed without merging**